### DISCUSSION TITLE
Discussion on supported versions

### DISCUSSION THREAD

**Discussion by panyogesh** [magma/magma#15289] [Communications]
This is regarding the version to be supported on Magma. 

As of today the released build is 1.8 and the next targeted release 1.9. Idea for this discussion is to get feedback on what is the last supported version to be officially supported. Here are few of the challenges :
1. 1.6 Officially was released on Debian.
2. Older version might have older OVS packages
3. Python and other dependent packages might get updated. 

So the idea is to support only the released version. So it means as soon as 1.9 is out that will be only supported version.

Let me know if there are any comments regarding the same.

**Comment by jordanvrtanoski** [comment]
We would need to support at minimum 2 versions:
- Current production version
- Current development version

**Comment by panyogesh** [comment]
#15127 : Rules for maintenance
